### PR TITLE
Add Cricket Stream

### PR DESCRIPTION
**Pull request by mberk**

### CODE REVIEW

File: betfairlightweight/streaming/betfairstream.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ class BetfairStream:
         lambda: "stream-api.betfair.com",
         integration="stream-api-integration.betfair.com",
         race="sports-data-stream-api.betfair.com",
+        cricket="sports-data-stream-api.betfair.com",
```
[the remark this answers is not in the document]
Comment (mberk) [reply betfairlightweight/streaming/betfairstream.py]: Done

File: betfairlightweight/streaming/stream.py [hunk 2]
```diff
@@ -255,3 +255,31 @@ def _process(self, race_updates: list, publish_time: int) -> bool:
...
+class CricketStream(BaseStream):
+    _lookup = "cc"
+    _name = "CricketStream"
+
+    def _process(self, cricket_change_messages: list, publish_time: int) -> bool:
+        caches, img = [], False
+        for cricket_change_message in cricket_change_messages:
+            market_id = cricket_change_message["marketId"]
+            event_id = cricket_change_message["eventId"]
+            cricket_match_cache = self._caches.get(market_id)
```
[the remark this answers is not in the document]
Comment (mberk) [reply betfairlightweight/streaming/stream.py:269]: I like the consistency of market ID although it's not a strong preference

**Comment by mberk** [comment]
I have added tests based on the ones for the Race stream. An observation is that the example messages given in the Betfair documentation do not include a lot of fields and so the tests for streamingresources/TestCricketMatch could be improved with better test data

File: tests/test_historicalstream.py [hunk 3]
```diff
@@ -87,3 +87,48 @@ def test_historical_generator_stream(self):
...
+        trading = betfairlightweight.APIClient("username", "password", app_key="appKey")
+        stream = trading.streaming.create_historical_generator_stream(
+            file_path="tests/resources/historicaldata/CRICKET-1.179676557",
+            listener=StreamListener(lightweight=True),
+            operation="cricketSubscription",
+        )
+        gen = stream.get_generator()
+        data = [i[0] for i in gen()]
+
+        print(data)
```
Comment (liampauling) [inline tests/test_historicalstream.py:127]: Are you able to remove this?

Comment (mberk) [reply tests/test_historicalstream.py]: Done

**Pull request merged**